tf/train.py

```diff
@@ -408,6 +408,9 @@ def main(cmd):
     # Load data with split batch size, which will be combined to the total batch size in tfprocess.
     ChunkParser.BATCH_SIZE = split_batch_size
 
+    value_focus_min = cfg['training'].get('value_focus_min', 1)
```

Comment:
Probably should throw an exception if these are non default values and experimental parser is enabled.